platform/lib/routers/growPages.js

```diff
@@ -275,7 +275,14 @@ growPages.get(/^(.*\/)?([^\/\.]+|.+\.html|.*\/|$)$/, async (req, res, next) => {
 
   let renderedTemplate = null;
   try {
-    renderedTemplate = template.render(templateContext);
+    renderedTemplate = await new Promise((resolve, reject) => {
```

Comment:
why do you need to wrap render into a promise?

Reply:
Needed since we switched to asynchronous template rendering.

Reply:
got it, you can also use Node's promisify helper for this as well. Why are we switching to async here?

Reply:
Async filters like the newly introduced `importBlog` only work when nunjucks is used in async mode which is implictly invoked by using `Template.render` with a callback.

Reply:
thanks for the clarification

Reply:
Still, @robinvanopstal please refactor to `promisify` as Sebastian suggested 😉